README.md

```diff
@@ -189,3 +190,9 @@ Once you have the images built you can use them in the previous commands by repl
 
 For a more sophisticated deployment which includes bidirectional header sync, message passing,
 monitoring dashboards, etc. see the [Deployments README](./deployments/README.md).
+
+## Community
+
+If you are interested in information exchange and development of Polkadot related bridges please feel free to join the [Polkadot Bridges](https://app.element.io/#/room/#bridges:web3.foundation) Element channel.
+
+The [Substrate Technical](https://app.element.io/#/room/#substrate-technical:matrix.org) Element channel is most suited for discussions regarding Substrate itself.
```

Comment:
Can you cut these lines off at 100 characters? That's what we use in the [.editorconfig](https://github.com/paritytech/parity-bridges-common/blob/master/.editorconfig) to keep things more readable in editors.

I'd also add a note about what Element is and a link to their website for anyone not familiar with it.